dictionaries/en_shared/src/shared-additional-words-ise.txt

```diff
@@ -0,0 +1,10 @@
+# These are shared words for all variants of English that uses the -ise suffix.
+# So mostly all English flavors except American English
+
+computerised
+parameterise
+serialisation
+synchronisation
+synchronise
+unmanoeuvrable
```

Comment:
I think it is better to have this be a `refactor` PR instead of a feat.

If you leave out the words, then the content of the dictionaries should not change.

Reply:
I thought ` refactor` was not allowed (not listed in GH template), is that OK to use it, or should I use `chore`?

Reply:
Never mind. It looks like these words are already in all those dictionaries.

Reply:
I'm fine with `refactor` it will work just fine. I tried to keep the GH template as minimal as possible to cover 90% of the cases.

Reply:
> Never mind. It looks like these words are already in all those dictionaries.

Yes, the feature I brought was only about creating the file, the words will be removed by the other PR to avoid conflicts